ticdc/manage-ticdc.md

```diff
@@ -208,11 +208,10 @@ The following are descriptions of parameters and parameter values that can be co
 | :------------------ | :------------------------------------------------------------ |
 | `127.0.0.1`          | The IP address of the downstream Kafka services                                 |
 | `9092`               | The port for the downstream Kafka                                          |
-| `topic-name`         | Variable. The name of the Kafka topic                                      |
```

Comment:
I don't quite understand why we need to remove it? cc: @3AceShowHand

Reply:
yes, no need to remove it, we should keep it.